app/component/map/NearYouMap.js

```diff
@@ -141,6 +141,7 @@ function NearYouMap(
     position,
     showWalkRoute,
     prioritizedStopsNearYou,
+    setMWTRef,
```

Comment:
Passing state mutating function is sometimes considered bad practice but since the ancestor function in `NearYouPage` looks like
```javascript
setMWTRef = ref => {
  this.MWTRef = ref;
};
```
without even having MWTRef as a field, I think this one _may_ be OK.